crypto/pedersen/pedersen_test.go

```diff
@@ -0,0 +1,25 @@
+// +build pedersenunit
+
+package pedersen
+
+import (
+	"testing"
+
+	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHashPayload(t *testing.T) {
```

Comment:
Let us also test for predefined payload and hash and compare the hashes instead?